tests/performance/vectorize_aggregation_combinators.xml

```diff
@@ -1,4 +1,4 @@
-<test>
+<test max_ignored_relative_change="0.1">
```

Comment:
This is the default. Both of these tests are not counted as errors, just show in the table so that we can see smaller changes.The errors were about the slow tests https://clickhouse-test-reports.s3.yandex.net/23319/ac68dd8e979fce7d764e566a1cae41efe9611663/performance_comparison/report.html#error-summary